### PR TITLE
docs(DIST-1468): Update docs with custom icon

### PR DESCRIPTION
**Pull request by antoniofull**
fix for #440

### CODE REVIEW

File: packages/embed/README.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ Closing and opening a typeform in modal window will restart the progress from th
...
 | enableSandbox          | boolean          | enable [sandbox mode](https://help.typeform.com/hc/en-us/articles/360029295952) (disables submissions and tracking)                                                                                               | `false`                                                       |
 | buttonText             | string           | customize the button text (sidetab only)                                                                                                                                                                          | `"Launch me"`                                                 |
+| customIcon             | string           | customize the message icon (popover, sidetab) )                                                                                                                                                                   | `"Image url"`                                                 |
```
Comment (mathio) [inline packages/embed/README.md:137]: The default value is not "image url" string.

**Review by mathio — CHANGES_REQUESTED** [review]

**Comment by mathio** [comment]
Please add dev portal docs to this repo ([see docs/configuration.md](https://github.com/Typeform/embed/blob/main/docs/configuration.md)), do not open [PRs](https://github.com/Typeform/dev-portal-content/pull/760) manually.

**Review by maxprilutskiy — APPROVED** [review]

**Review by mathio — CHANGES_REQUESTED** [review]
Can we please try to keep those 2 tables ([README](https://github.com/Typeform/embed/blob/dd4333c27ad3df37bc1880c1f6c42ae7fa5eb316/packages/embed/README.md) and [configuration docs](https://github.com/Typeform/embed/blob/dd4333c27ad3df37bc1880c1f6c42ae7fa5eb316/docs/configuration.md)) in sync? Same order of things, same text.

**Comment by antoniofull** [comment]
> Can we please try to keep those 2 tables ([README](https://github.com/Typeform/embed/blob/dd4333c27ad3df37bc1880c1f6c42ae7fa5eb316/packages/embed/README.md) and [configuration docs](https://github.com/Typeform/embed/blob/dd4333c27ad3df37bc1880c1f6c42ae7fa5eb316/docs/configuration.md)) in sync? Same order of things, same text.

I don't see the order, buttonText in the configuration?

**Comment by mathio** [comment]
> I don't see the order, buttonText it's not in the configuration?

Good catch, lets sync (copy-paste) those two tables. I just pushed a commit with this.

**Comment by antoniofull** [comment]
Thanks @mathio feel free to re-approve and merge it when ready

**Review by mathio — APPROVED** [review]

**Pull request merged**